.github/workflows/schema_update.yml

```diff
@@ -0,0 +1,114 @@
+name: Update GraphQL schema in Apollo Studio
+on:
+  push:
+    branches:
+      - main
+
+env:
+  APP_NAME: ${{ github.event.repository.name }}
+  APP_PORT: "3000"
+  APP_IMAGE_TAG: ${{ github.sha }}
+  DOCKER_NETWORK: ci
+  DATABASE_URL: "postgres://postgres:holaplex@db:5432/hub_nfts"
+  KAFKA_BROKERS: "redpanda:9092"
+  APOLLO_PROJECT: hub-dev@current
+
+jobs:
```

Comment:
We have docker-compose that is capable of starting up services need to start up the webserver.

I found this github action or look to use`docker compose up`

Reply:
thought about it but in this setup we are aiming for container to container connectivity, and our docker compose is meant to be used with local service -> container

Reply:
Does it work if docker compose given a network then run this on the same network?

https://github.com/holaplex/hub-nfts/pull/46/files#diff-478656722c5288a0fec76bc23438f8ed204d922c307a1b07bf099108d58528edR64

Reply:
Usually yes, but the problem here is that you need to tell redpanda what the OUTSIDE hostname is on startup.
In the docker-compose file we use `localhost`, but [here](https://github.com/holaplex/hub-nfts/pull/46/files#diff-478656722c5288a0fec76bc23438f8ed204d922c307a1b07bf099108d58528edR47) we need to use the container name.
If you dont use the specified IP/HOST connection will fail.